pyproject.toml

```
@@ -0,0 +1,123 @@
+[build-system]
+requires = ['setuptools', 'wheel', 'pytest-runner>=2.11.1']
```

So this isn't ideal (putting pytest-runner here). This is meant to be the minimal dependencies to just build the library (make whl/tar.gz file). So it shouldn't include the items to run the tests. 
Does it break it if you remove it? 
```suggestion
requires = ['setuptools', 'wheel']
```

---

Yes this is what I read also haha, I will revert it ;), thanks @gsheni